### DISCUSSION TITLE
[sly] Failed to connect to Slynk: inferior process exited.

### DISCUSSION THREAD

**Discussion by daninus14** [joaotavora/sly#654] [Q&A] [answered]
Hi, I'm getting this error when trying to start sly.

Any ideas on how to troubleshoot?

**Comment by joaotavora** [comment]
Look in the `*inferior-lisp*` buffer for clues.

**Reply by joaotavora** [reply]
Actually called `*sly-started inferior-lisp....*` or something like that.

**Reply by aadcg** [reply, marked as answer]
`*sly-inferior-lisp for <lisp-implementation>*`

**Reply by daninus14** [reply]
Thanks! Will do

**Reply by daninus14** [reply]
Thanks again. 

I just want to note for anyone (myself include) in the future that if `C-x b` doesn't show you the buffer, doing `C-x C-b` will probably show you the buffer and you can hit enter on the buffer name in the list to open it.

Also in my case the buffer that had useful information was `*sly-started inferior-lisp for qlot*`. 
`*sly-inferior-lisp for sly*` Was actually empty.
